apps/aestratum/src/aestratum_env.erl

```diff
@@ -0,0 +1,48 @@
+-module(aestratum_env).
+
+%% ENV handling
+-export([setup/0,
+         get/1,
+         get/2,
+         set/1,
+         set/2,
+         reset/1,
+         unset/0, unset/1,
+         keys/0]).
+
+%%%%%%%%%%
+
+setup() ->
+    unset(),
+    UserConfig = aeu_env:user_config(<<"stratum">>, [{<<"enabled">>, false}]),
```

Comment:
Did you mean
```suggestion
    UserConfig = aeu_env:user_config([<<"stratum">>, <<"enabled">>], false),
```
?

Reply:
I think it's good as it is - if there's NO stratum config at all (not even: `stratum -> enabled -> false`) it behaves as it's disabled.
Allows those users which are not interested in stratum to upgrade to a newer version of node without touching of configuration file.